packages/cinterop/build.gradle.kts

```diff
@@ -206,37 +206,36 @@ kotlin {
                 implementation("androidx.test:rules:${Versions.androidxTest}")
             }
         }
-        val macosMain by getting {
-            // TODO HIERARCHICAL-BUILD From 1.5.30-M1 we should be able to commonize cinterops using
-            //  kotlin.mpp.enableCInteropCommonization=true (https://youtrack.jetbrains.com/issue/KT-40975)
-            //  This would also require us to enable hierarchical setup, which is currently blocked by
-            //  https://youtrack.jetbrains.com/issue/KT-48153
-            // FIXME HIERARCHICAL-BUILD Rename to nativeDarwin
-            kotlin.srcDir("src/darwin/kotlin")
+        val nativeDarwin by creating {
```

Comment:
Aren't this one created automatically? https://kotlinlang.org/docs/multiplatform-share-on-platforms.html#share-code-in-libraries

This would also impact a lot of wiring below in this file.

Update:
After trying to build without creating this, I can see it is failing, which I guess is why you are creating it and after re-reading the documentation it became clear that the above link was just an example, and the only supported shortcuts are these ones 🙈 https://kotlinlang.org/docs/multiplatform-share-on-platforms.html#use-target-shortcuts

Reply:
🤪  ... 👍